### DISCUSSION TITLE
What ticks per second do the official servers run on?

### DISCUSSION THREAD

**Discussion by AbdelRahmanYaghi** [ValveSoftware/Dota2-Gameplay#6777] [General]
I know that custom games run on 30 tps. But what about matchmaking servers?

**Comment by jackluli** [comment]
Matchmaking servers for most popular games typically run at 60 tps, which is twice as fast as the 30 tps used for custom games. However, this can vary depending on the specific game and its server configuration. Some games may use lower or higher tps depending on their requirements and performance. It's important to note that a higher tps can provide a smoother and more responsive gameplay experience, but it also requires more processing power and network bandwidth, which can impact server performance and stability[.](http://goldwa.dev/)

**Reply by Lesternixon** [reply]
The matchmaking servers for the majority of well-known games normally operate at 60 tps, which is twice as fast as the 30 tps required for bespoke games. The unique game and its server settings will determine how this changes. Depending on their needs and capabilities, some games may employ lesser or higher tps. It's vital to remember that while a higher tps may offer a smoother and more responsive gaming experience, it also necessitates more computing power and network capacity, which may affect server performance and stability(.)(https://overplugged.org/)

**Comment by run3unblockedaz** [comment]
Excellent and intriguing essay. Thank you very much! Thank you for sharing this outstanding content. <a href="https://run3donline.com/"> **run 3**</a> is a blog that will help you recover your password and regain access to your account if you've lost it.

**Comment by Lesternixon** [comment]
[https://overplugged.org/](url)

**Reply by Lesternixon** [reply]
Copriletti estivi are an essential item to have for summer months. Whether you're looking for a lightweight blanket to keep you cool in the heat or a stylish coverlet to add a touch of class to your bedroom, copriletti estivi offer the perfect solution. Not only are they available in a variety of colors and prints, but they provide maximum comfort and breathability while keeping out the heat. With so many options available, it's easy to find one that suits your taste and budget. Explore copriletti estivi today for an easy way to update your bedroom and enjoy a comfortable night’s sleep all summer long(copriletti estivi)(https://pettihome.shop/collections/copriletti-estivi)

**Comment by KESIE07** [comment]
Exceptional and fascinating essay. I sincerely appreciate it. I appreciate you giving this fantastic information. If you've forgotten your password,[ myfedloan](https://www.myfedloan.us/)'s blog will assist you in getting it back and regaining access to your account.

**Comment by Lesternixon** [comment]
Our beloved Iced out heart ring is stunning, sparkly, and popular! Ideal for wearing alone or with other rings. There are three sizes available in gold and silver[.](https://luxovany.com/products/iced-out-heart-ring)

**Comment by ZamirZen** [comment]
It’s great to delve into the technical aspects of the game server. Although standard games run around 30 ticks per second (tps), matchmaking servers may have different settings. Generally, matchmaking servers operate at a higher tick rate making gameplay easier and more efficient. Tick ​​rates can vary, but are usually 64 or 128 ticks per second, allowing for accurate communication and quick updates.

Understanding the technology behind it can really enhance your gaming experience. Keep those questions coming, and happy playing!

<a href="https://instaproapps.net/" style="color:#FFFFFF;">insta pro</a>
